v1.0/deploy-cockroachdb-to-production.md

```diff
@@ -0,0 +1,424 @@
+
+For details about controlling the number and location of replicas, see [Configure Replication Zones](configure-replication-zones.html).
+
+## Deployment Methods
+
+Because of CockroachDB's cloud-native design, it works best in conjunction with an **orchestration tool**, such as Kubernetes.
+
+It is certainly possible to run CockroachDB manually, but tasks such as monitoring node liveliness and scaling your cluster are handled much more efficiently with an orchestration tool.
+
+### Load Balancing
+
+If you opt to run CockroachDB outside of an orchestration tool, you will also need to be sure to configure a load balancer manually. This include setting up things like service discovery to ensure that you are aware when new nodes are added to your cluster.
+
+### Service Discovery
+
+If you choose to deploy CockroachDB without an orchestration tool, you'll also need to manually set up a service discovery tool, such as Zookeeper.
+
+At this time, CockroachDB doesn't offer native integration with any service discovery tools, so you will need to manually register nodes with your service discovery tool after starting your node.
+
+### Monitoring
+
+While an orchestration tool can automatically bring up new resources if machines fail, if you choose to deploy CockroachDB manually, you'll need to integrate it with a monitoring tool to alert your team of node failures.
+
+You can monitor your cluster with basically any tool, but we have documentation on integrating CockroachDB with [Prometheus](monitor-cockroachdb-with-prometheus.html).
+
+## Security
+
+To deploy CockroachDB, you *must* use SSL encryption. To make this as simple as possible, the `cockroach` binary can work a certificate authority (CA), which provides everything you need to generate SSLs.
+
+However, CockroachDB supports using any valid certificates, such as those generated by OpenSSL.
+
+Regardless of which option you choose, you'll need the following files:
+
+- A certificate authority (CA) certificate and key, used to sign all of the other certificates
+- A separate certificate and key for each node in your deployment, with the common name `node`
+- A separate certificate and key for each client and user you want to connect to your nodes, with the common name set to the username. The default user is `root`.
+  
+  Alternatively, CockroachDB does [support password authentication](create-and-manage-users.html#secure-cluster), though we typically recommend using client certificates instead.
+
+### Deploying Certificates
+
+#### Nodes
+
+Each node requires local copies of the following:
+
+- Your certificate authority (CA) certificate to validate the signature of all other certificates
+- A node key and certificate signed by your CA with the common name of `node`
```

Comment:
Should probably mention the ip/dns addresses that can be used to reach the nodes.